server/src/main/java/org/tctalent/server/model/db/TaskDtoHelper.java

```diff
@@ -53,8 +51,7 @@ public boolean ignoreProperty(Object o, String property) {
                 questionOnlyProperties.contains(property) &&
                     ! (o instanceof QuestionTask || o instanceof QuestionTaskAssignment) ||
                 uploadOnlyProperties.contains(property) &&
-                    ! (o instanceof UploadTask || o instanceof UploadTaskAssignment);
-
+                        !(o instanceof UploadTask || o instanceof UploadTaskAssignment);
```

Comment:
Minor one, keep an eye on any accidental indentations. Sometimes your Intellij may be doing it without you knowing, I can share my intellij config if you don't already have it as we try to have same config across team.